cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KRaftMetadataManager.java

```diff
@@ -63,28 +63,25 @@ public class KRaftMetadataManager {
     public static Future<Void> maybeUpdateMetadataVersion(
             Reconciliation reconciliation,
             Vertx vertx,
-            SecretOperator secretOperator,
+            TlsPemIdentity kafkaTlsPemIdentity,
```

Comment:
Same as above - I won't repeat the comment for other places -> this should be the identity of who connects. So CO, or?

Reply:
I've changed them to coTlsPemIdentity across the board